issue_15/script.js

```diff
@@ -0,0 +1,87 @@
+const showBtn = document.getElementById('js-show-btn');
+const modal = document.getElementById('js-modal');
+const closeBtn = document.getElementById('js-close-btn');
+const submitBtn = document.getElementById('js-submit-btn');
+
+
+function startLoading() {
```

Comment:
IMO
```suggestion
function renderLoading() {
```

Reply:
Thank you for comments.
Fixed:sparkles:
218fd7e2fa5470fa1d790a57284f49737ce4ab4d